HadesScheduler/main_test.go

```
@@ -0,0 +1,47 @@
+// the fields would silently stay at their zero values. NewHadesConsumer's
+// withDefaults() would still run the consumer correctly, so nothing would
+// break at runtime; the damage would be confined to the startup log reporting
+// 0 / 0s / 0 while the consumer actually used 1 / 1m / 3. A configuration log
+// that disagrees with the effective configuration is worse than no log.
+func TestConsumerConfigDefaultsAreLoaded(t *testing.T) {
+	// Clear rather than set: t.Setenv with an empty string is still a set
+	// variable, and an explicit empty value is not the same as an absent one.
+	// Restored by t.Cleanup so the test does not leak into its neighbours.
+	for _, key := range []string{"CONCURRENCY", "NATS_ACK_WAIT", "NATS_MAX_DELIVER"} {
+		if old, ok := os.LookupEnv(key); ok {
+			t.Cleanup(func() { os.Setenv(key, old) })
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)